utils/addFeatureFlag.ts

```diff
@@ -0,0 +1,73 @@
+import { ObjectLiteralExpression, PropertyAssignment, SourceFile, SyntaxKind } from "ts-morph";
+import { addPropertyToObject } from "./tsMorph/addPropertyToObject";
+import { getDefaultExportedObject } from "./tsMorph/getDefaultExportedObject";
+
+export type AddFeatureFlagResult = {
+    success: boolean;
+    hasError: boolean;
+};
+
+const FEATURE_FLAGS_PROP_NAME = "featureFlags";
+
+/*
+ * Add feature flag property to existing featureFlags object. In case featureFlags property
+ * doesn't exist, the property and object will be created and feature flag will be set.
+ * */
+export const addFeatureFlag = (
+    source: SourceFile,
+    name: string,
+    value: Record<string, any> | string | boolean
+): AddFeatureFlagResult => {
+    if (!source) {
+        return {
+            success: false,
+            hasError: true
```

Comment:
What's the value of `hasError` here? I mean, if we return `success: false`, then it's logical that `hasError` is true. I can determine if there was an error from the `success` flag itself.

Let's rather just throw errors if something is wrong. Then we catch the error in the main file, and print the error msg.